storagenode/storagenodedb/bandwidthdb.go

```diff
@@ -197,6 +182,7 @@ func (db *bandwidthdb) Rollup(ctx context.Context) (err error) {
 		SELECT datetime(strftime('%Y-%m-%dT%H:00:00', created_at)) created_hr, satellite_id, action, SUM(amount)
 			FROM bandwidth_usage
 		WHERE datetime(created_at) < datetime(?)
+		AND datetime(created_at) >= coalesce((SELECT datetime(MAX(interval_start), '+1 hour') FROM bandwidth_usage_rollups), datetime(0, 'unixepoch'))
 		GROUP BY created_hr, satellite_id, action;
 
 		DELETE FROM bandwidth_usage WHERE datetime(created_at) < datetime(?);
```

Comment:
This would discard some values that weren't accounted before.

Reply:
so your concern is data from the past is added and then not rolled up? I think that is the expectation of rolling up, otherwise why roll the data?

Reply:
Yes, data from past is added and then discarded without rolling them up.

Reply:
correct I think that's intentional with this system

Reply:
removed in favor of new insert/on conflict/transaction code